### ISSUE TITLE
Explore elimination of edit wrappers

### ISSUE THREAD

**Issue by chrisvxd** [measuredco/puck#455]
Puck introduces various wrappers during edit mode to provide drag-and-drop and edit functionality. This can result in different interface between edit and render modes, such as:

* `children` containing different nodes
* CSS Grid styles across DropZones not being possible in edit mode, even if it works in render mode
* Inline elements do not render correctly in edit mode (#4)

We should explore ways to eliminate these styles without compromising the Puck interface. A solution may or may not involve breaking changes.

**Comment by sagarchoudhary96** [comment]
@chrisvxd any updates on this one ?

**Comment by chrisvxd** [comment]
Yes - it's currently in progress as part of efforts to support #123, which is requiring a drag-and-drop engine rebuild. Will update the status.
